unified_range/api.py

```diff
@@ -16,13 +16,13 @@ def from_semver(semver_spec: str) -> str:
     return str(ver_rng)
 
 
-def to_semver(spec: str) -> str:
+def to_semver(spec: str, comma_separated: bool=False) -> str:
```

Comment:
We need to think if it's better to have another api for that. or give the optional default arg on `to_semver`.

Reply:
has a default value of `" "`, maintaining old behavior and code.